Backend/profile/profile.go

```diff
@@ -98,6 +97,18 @@ func GetOneUserByUsername(username string) models.User {
 	return returnUser
 }
 
+func FindUserByFullName(firstName string, lastName string) models.User {
+	var returnUser models.User
+
+	err := db.QueryRow("SELECT id, first_name, last_name, username, bio FROM users WHERE first_name = ? AND last_name = ?", firstName, lastName).Scan(&returnUser.Id, &returnUser.FirstName, &returnUser.LastName, &returnUser.Username, &returnUser.Bio)
```

Comment:
So, it searches the database for the user where the first_name and last_name are the parameters of the function. Then, it "scans" the row, and it matches the values to the returnUser object. so the "id" that returns from the SQL query gets put into the retrunUser.Id, the "first_name" from the sql query gets put into returnUser.FirstName, and so on. I believe it's just based on the order, so since id appears in the SQL query first, I had to put &returnUser.Id first to make sure the "Id" value would be placed in the correct variable